gui/Cargo.toml

```diff
@@ -23,9 +36,18 @@ uuid = { version = "1.10.0", features = ["serde", "v4"] }
 
 [target.'cfg(target_arch = "x86_64")'.dependencies]
 x86-64 = { path = "../arch/x86-64" }
+raw-window-handle = "0.6"
+clap = { version = "4.5.20", features = ["derive"], optional = true }
+
+[dependencies.slint]
+git = "https://github.com/slint-ui/slint"
+rev = "875ca075fb5b2dfe4c3ab0a499d5759412fc1395"
```

Comment:
Is it possible to use the published version here? The same reason as `cbindgen`.

Reply:
Not at the moment

Reply:
In that case we should create an issue for this so when the release we need available we can fix this.